l2tdevtools/build_helpers/rpm.py

```diff
@@ -440,23 +440,23 @@ def __init__(self, project_definition, l2tdevtools_path):
       self.architecture = 'noarch'
 
   def _GenerateSpecFile(
-      self, project_name, project_version, source_filename,
+      self, project_name, project_version, source_package_filename,
       source_helper_object):
     """Generates the rpm spec file.
 
     Args:
       project_name (str): name of the project.
       project_version (str): version of the project.
-      source_filename (str): name of the source package file.
+      source_package_filename (str): name of the source package file.
       source_helper_object (SourceHelper): source helper.
 
     Returns:
       str: path of the generated rpm spec file or None if not available.
     """
     source_directory = source_helper_object.Create()
     if not source_directory:
-      logging.error(
-          'Extraction of source package: {0:s} failed'.format(source_filename))
+      logging.error('Extraction of source package: {0:s} failed'.format(
```

Comment:
Please include the source_directory in the log message here as well.

Reply:
Same comment as before, see line 457, `source_directory` is "empty"